src/app/services/book.service.ts

```diff
@@ -1,16 +1,29 @@
 import {Injectable} from '@angular/core';
-import {Http, Response} from '@angular/http';
+import {Http, Headers} from '@angular/http';
 import {Observable} from 'rxjs/Observable';
+import {Book} from '../models/book.model';
 
 @Injectable()
 export class BookService {
+  private headers = new Headers({'Content-Type': 'application/json'});
 
-  private getBooksUrl = 'http://milenabooks.azurewebsites.net/api/Books';
+  private getPostBooksUrl = 'http://milenabooks.azurewebsites.net/api/Books';
 
   constructor(private http: Http) {
   }
 
-  getBooks(): Observable<Response> {
-    return this.http.get(this.getBooksUrl);
+  getBooks(): Observable<Book[]> {
+    return this.http.get(this.getPostBooksUrl)
```

Comment:
You use the md-paginator in the component, but you haven't implemented pagination. Please, consider implementing pagination so that this feature is complete.

Another comment regarding the layout (Just putting it here because there are no changes in the layout).  I'd recommend using cards, since it's not really appropriate to use a table in the use case, especially considering that the book model contains a high-res image of a book cover.

The paginator is great, but I think it's better to use cards for listing book instead of table rows.